commands/hubdb/fetch.ts

```diff
@@ -48,7 +58,7 @@ exports.builder = yargs => {
   addConfigOptions(yargs);
   addUseEnvironmentOptions(yargs);
 
-  yargs.positional('tableId', {
+  yargs.positional('table-id', {
```

Comment:
Should this become an option now? `yargs.option()`

Reply:
It didn't say to turn it into an `option` in the issue, but does falling back to prompts imply that the positionals become options?